common/kommander-operator/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- helmrelease.yaml
```

Comment:
This `kommander-operator` folder wont be available when we release kommande-application unless you include it via [release command   ](https://github.com/mesosphere/kommander-applications/blob/1cc697222be1f34d87abe024dcbcd006ad0d5d42/make/release.mk#L11)

Reply:
It's already in common, right?

Reply:
oh i see